src/cmdc_tools/datasets/official/NV/data.py

```diff
@@ -17,14 +17,18 @@ class NevadaFips(DatasetBaseNoDate):
     def get(self):
         cases = self._get_cases()
         tests = self._get_tests()
-        return pd.concat([tests, cases], sort=False)
+        hosp = self._get_hosp()
+        return pd.concat([tests, cases, hosp], sort=False)
```

Comment:
Let's also add `ignore_index=True` as a keyword argument so we get a proper index